src/lava/proc/dense/models.py

```diff
@@ -188,3 +188,148 @@ def run_spk(self):
         )
 
         self.recv_traces(s_in)
+
+
+class AbstractPyDelayDenseModel(PyLoihiProcessModel):
+    """Abstract Conn Process with Dense synaptic connections which incorporates
+    delays into the Conn Process.
+    """
+
+    def get_del_wgts(self):
```

Comment:
Can we add a test for this weight & delay matrix expansion to tensor?

Reply:
Done. Also made the method a static method.